godel_path_planning/CMakeLists.txt

```diff
@@ -7,10 +7,18 @@ find_package(catkin REQUIRED COMPONENTS
   godel_process_path_generation
   roscpp
   cmake_modules
+  descartes_core
+  descartes_moveit
+  descartes_planner
+  descartes_trajectory
+  moveit_ros_move_group
```

Comment:
Are we linking against anything in "moveit_ros_move_group"? if not then it's probably not needed in this list.